prj.conf

```diff
@@ -10,6 +10,10 @@ CONFIG_MAIN_STACK_SIZE=8192
 CONFIG_POSIX_API=y
 CONFIG_REBOOT=y
 
+#### System Clock Configuration ####
+# Run system clock at 1 MHz for microsecond precision timing
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000000
```

Comment:
This change will have a massive effect on the system. It will change interrupt timings, thread switching timings, and any other timings defined by the system. It will also increase overhead of kernel operations (context switching, etc) because these will happen more often.

Ticks represent the minimum time quantum that the kernel can detect and act on.  We currently use `10000`.  This increases that value by `100x`.